stablehlo/reference/Element.cpp

```diff
@@ -601,6 +601,25 @@ Element atan2(const Element &e1, const Element &e2) {
                                      debugString(type).c_str()));
 }
 
+Element cbrt(const Element &el) {
+  auto type = el.getType();
```

Comment:
Seems like a candidate to use `mapWithUpcastToDouble` https://github.com/openxla/stablehlo/blob/da30def7b19529899c354f5527a0cbe61a5e45b6/stablehlo/reference/Element.cpp#LL103C9-L103C30